### DISCUSSION TITLE
Change the blue color where the user avatar is displayed.

### DISCUSSION THREAD

**Discussion by bonecountysheriff** [Docile-Alligator/Infinity-For-Reddit#997] [Completed Ideas] [answered]
I don't know what that location is called, but there seems to be no option in the theme settings to change that color. Can you help me out?

**Comment by Wladefant** [comment]
Settings>theme>your theme under customization>username color

**Comment by bonecountysheriff** [comment]
![IMG_20220815_235011](https://user-images.githubusercontent.com/39475970/184692919-59fdcaf3-8679-436a-83ba-84824bca32ea.jpg)
This blue background is what I meant. The setting you mentioned changes the username font color.

**Reply by Wladefant** [reply]
Sorry, misunderstood you

**Comment by bbaster** [comment, marked as answer]
Seems to be introduced with commit 275063684779a02efe7ec8ef971bb219b1f4fd33
